kickstart/create-game.yaml

```
@@ -0,0 +1,30 @@
+summary: Create the $SRYYYY game
+
+priority: should
+
+component: Rules
+
+milestone: $SRYYYY Kickstart
+
+area-owner: game
+
+description: >-
+    The $SRYYYY game will be different than the previous year's, so we
+    need to decide what it will be.
+
+    Note: once decided we still want to keep the choice a secret so it's
+    a surprise to teams at Kickstart.
+
+    Once the game is decided, we'll need to generate a set of rules for
+    it. This will cover the arena layout, the scoring, and the regulations
+    for the robots.
+
+    For the past few years we've also set a set of challenges for the teams
+    to complete during the year, to encourage them to begin working on their
+    robots early. These include using the vision system, performing accurate
+    movements, and a mechanism challenge.
+
+    Required actions:
+    - [ ] Decide on the game
+    - [ ] Write the rules document
```

In the past, explicitlty getting input from volunteers on mass has been incredibly helpful in finding the weird edge cases in the rules.

```suggestion
    - [ ] Write the rules document
    - [ ] Invite mass review of the rules from volunteers
```